internal/config/config.go

```diff
@@ -168,3 +183,12 @@ func parseBool(input string) bool {
 	_, err := strconv.ParseBool(input)
 	return err == nil
 }
+
+func parseIP(input string) string {
+	ip := net.ParseIP(input)
+	if ip == nil {
+		panic("invalid IP address: " + input)
```

Comment:
suggestion: quote so users can see issues like trailing white space
```suggestion
		panic(fmt.Sprintf("invalid IP address: %q", input))
```